app/assets/stylesheets/posts.scss

```diff
@@ -1,3 +1,113 @@
 // Place all the styles related to the Posts controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+.gallery {
+  margin-top: 3%;
+  max-height: 700px;
+  &.container {
+    background-color: white;
+    .row [class*="col-"] {
```

Comment:
for this reason better to add classes `no-margin`, `no-padding` and use in layout